plutip-server/Types.hs

```diff
@@ -99,14 +100,15 @@ instance FromJSON Lovelace where
       then fail "Lovelace value must not be negative"
       else pure $ Lovelace value
 
-data StartClusterRequest = StartClusterRequest
-  { slotLength :: NominalDiffTime
-  , epochSize :: EpochSize
-  , maxTxSize :: Natural
-  , increasedExUnits :: Bool
-  , -- | Lovelace amounts for each UTXO of each wallet
-    keysToGenerate :: [[Lovelace]]
-  }
```

Comment:
Would it be possible to change this to be one constructor, and make it so that any keys that are missing from the request (or are set to `Nothing`) are given default values? This would simplify things for users, as currently if they want to change one of these values they have supply values for all of them. It would also simplify (in my opinion) the design in your other PR ([see my review](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1334#pullrequestreview-1221919148))